### PR TITLE
Update the front-end package to use bootstrap 4.3.1

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes CVE-2019-8331

### CODE REVIEW

File: app/assets/javascripts/blacklight/blacklight.js [hunk 1]
```diff
@@ -28,16 +27,15 @@ Blacklight = function () {
       return listeners;
     }
   };
-}();
-
-// turbolinks triggers page:load events on page transition
+}(); // turbolinks triggers page:load events on page transition
```
Comment (cbeer) [inline app/assets/javascripts/blacklight/blacklight.js]: What's the relationship between `app/assets/javascripts` and `app/javascripts`? It seems like you've (or someone?) made some changes to one that aren't reflected in the other?

Comment (jcoyne) [reply app/assets/javascripts/blacklight/blacklight.js]: `app/assets/javascripts` is the compiled assets from `app/javascripts`. In a pure JS project the former would be `dist` and the latter `src`

Comment (jcoyne) [reply app/assets/javascripts/blacklight/blacklight.js]: I think the whitespace difference is from the updated version of babel.

Comment (cbeer) [reply app/assets/javascripts/blacklight/blacklight.js]: got it; it's made some really weird choices about where comments go, i guess.

**Pull request merged**